test/e2e/basics-e2e-specs.js

```diff
@@ -17,12 +18,12 @@ chai.use(chaiAsPromised);
 var LAUNCH_HANDLER_TIMEOUT = 10000;
 var TEMP_DIR = path.resolve(__dirname, 'tmp');
 
-describe('intruments tests', function () {
+describe('instruments tests', function () {
   this.timeout(90000);
 
   async function newInstrument(opts) {
     _.extend(opts, {
-      app: path.resolve(rootDir, 'test/assets/TestApp.app'),
+      app: path.resolve(__dirname, '../../..', 'node_modules', 'ios-test-app', testAppPath[1]),
```

Comment:
seems like we're using path.resolve inconsistently, even within this one call. would like to see either:

```js
path.resolve(__dirname, '..', '..', '..', 'node_modules', 'ios-test-app', testAppPath[1]);
```

or

```
path.resolve(__dirname, '../../../node_modules/ios-test-app', testAppPath[1]);
```

same goes for previous use of this function